traits/has_traits.py

```diff
@@ -2559,6 +2528,35 @@ def on_trait_change(
         you must explicitly specify the 'xxx_items' trait if you want to
         be notified of changes to any of the items of the 'xxx' trait.
 
+        Parameters
```

Comment:
Here moved `Parameters` to the end. Without this, Sphinx was trying to interpret everything that followed as part of the Parameters section, with ungood results.